### ISSUE TITLE
haunted without shadow DOM is not working on IE11

### ISSUE THREAD

**Issue by lsimone** [crisward/haunted-ie11#1]
If you just add the option `{ useShadowDOM: false }` your build results in an error (on IE11, but also on Chrome).

```
customElements.define('my-counter', component(Counter, { useShadowDOM: false }));
```

![image](https://user-images.githubusercontent.com/3266650/73017110-8258fe00-3e1f-11ea-88d1-0da1f9d52ea8.png)

Do you know what's breaking the compatibility?

**Comment by crisward** [comment]
Not really, this was just a quick experiment to see what it took to get Haunted working in IE. It's also using an older version of haunted so not 100% sure the latest release still works. Supporting web components in IE seems to be a up hill battle which I'm hoping will soon become unnecessary with Chromium Edges IE mode supplanting IE11.
